### DISCUSSION TITLE
is this library planing for far future suport android too?

### DISCUSSION THREAD

**Discussion by MatejMagat305** [gtk-rs/gtk4-rs#1349] [Q&A]
I m only asking whether you planing android app? if yes, maybe would be interasting https://github.com/cnlohr/rawdrawandroid for realy small apk ..., or you already have others plans?

**Comment by sdroege** [comment]
You'd have to bring that up with the GTK project first of all. https://gitlab.gnome.org/GNOME/gtk/

Once/if GTK supports Android the bindings can also support it.
